launch

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+build() {
+    mkdir -p build
```

Comment:
Do we specifically not want to build in /tmp?

Reply:
Since this is launched from the repository folder, it won't be built at /tmp...

Reply:
(unless what you mean is that we want to build it at /tmp...)

Reply:
I am asking if we should do it the same as currently written in HACKING.md [here](https://github.com/snapcore/snapd/pull/12620/files#diff-9769a3f5504af3cd735e08f078b94f827bdd51eacac2e9d80ac9761bc485171fR206)

Reply:
Mmm... good point